gap/PolygonalComplexes/navigation.gd

```diff
@@ -1082,4 +1082,210 @@ DeclareOperation("FacesWithEdgeProperties", [IsVEFComplex, IsList] );
+#! [ 3, 5 ]
+#! @EndExampleSession
+#! 
+#! @Returns a set of positive integers
+#! @Arguments complex, face1, face2
+DeclareOperation( "CommonVerticesOfFaces", [IsVEFComplex, IsPosInt, IsPosInt]);
+#! @Arguments complex, faceList
+DeclareOperation( "CommonVerticesOfFaces", [IsVEFComplex, IsList]);
+#! @Arguments complex, face1, face2
+DeclareOperation( "CommonVerticesOfFacesNC", [IsVEFComplex, IsPosInt, IsPosInt]);
+#! @Arguments complex, faceList
+DeclareOperation( "CommonVerticesOfFacesNC", [IsVEFComplex, IsList]);
+#! @EndGroup
+
+#! @BeginGroup CommonEdgesOfVertices
+#! @Description
+#! The method <K>CommonEdgesOfVertices</K>(<A>complex</A>, <A>vertex1</A>, <A>vertex2</A>) returns the
+#! set of all edges that are incident to <A>vertex1</A> and <A>vertex2</A>. The NC-version does
+#! not check whether the given <A>vertex1</A> and <A>vertex2</A> is a vertex of <A>complex</A>.
+#! 
+#! The method <K>CommonEdgesOfVertices</K>(<A>complex</A>, <A>vertexList</A>) returns the
+#! set of all edges that are incident to all the vertices in <A>vertexList</A>. The NC-version does
+#! not check whether the given vertices in <A>vertexList</A> are a vertex of <A>complex</A>.
+#! 
+#! As an example, consider the polygonal complex that was introduced at the
+#! start of section <Ref Sect="Section_Access_BasicAccess"/>:
+#! <Alt Only="TikZ">
+#!   \begin{tikzpicture}[vertexStyle, edgeStyle, faceStyle=nolabels]
+#!       \input{Image_StarOfStarExample.tex}
+#!   \end{tikzpicture}
+#! </Alt>
+#! @ExampleSession
+#! gap> CommonEdgesOfVertices(complex,3,5);
+#! [ 9 ]
+#! @EndExampleSession
+#! 
+#! @Returns a set of positive integers
+#! @Arguments complex, vertex1, vertex2
+DeclareOperation( "CommonEdgesOfVertices", [IsVEFComplex, IsPosInt, IsPosInt]);
+#! @Arguments complex, vertexList
+DeclareOperation( "CommonEdgesOfVertices", [IsVEFComplex, IsList]);
+#! @Arguments complex, vertex1, vertex2
+DeclareOperation( "CommonEdgesOfVerticesNC", [IsVEFComplex, IsPosInt, IsPosInt]);
+#! @Arguments complex, vertexList
+DeclareOperation( "CommonEdgesOfVerticesNC", [IsVEFComplex, IsList]);
+#! @EndGroup
+
+#! @BeginGroup CommonFacesOfEdges
+#! @Description
+#! The method <K>CommonFacesOfEdges</K>(<A>complex</A>, <A>edge1</A>, <A>edge2</A>) returns the
+#! set of all faces that are incident to <A>edge1</A> and <A>edge2</A>. The NC-version does
+#! not check whether the given <A>edge1</A> and <A>edge2</A> is an edge of <A>complex</A>.
+#! 
+#! The method <K>CommonFacesOfEdges</K>(<A>complex</A>, <A>edgeList</A>) returns the
+#! set of all faces that are incident to all the edges in <A>edgeList</A>. The NC-version does
+#! not check whether the given edges in <A>edgeList</A> are an edge of <A>complex</A>.
+#! 
+#! As an example, consider the polygonal complex that was introduced at the
+#! start of section <Ref Sect="Section_Access_BasicAccess"/>:
+#! <Alt Only="TikZ">
+#!   \begin{tikzpicture}[vertexStyle=nolabels, edgeStyle, faceStyle=]
```

Comment:
There is an "=" after "faceStyle". This is probably the reason why the corresponding image is not found (the images are encoded by a hash of their source code).